hadoop-ozone/dist/src/main/compose/ozone-mr/test.sh

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )
+ALL_RESULT_DIR="$SCRIPT_DIR/result"
+set -x
+RESULT=0
+IFS=$'\n'
+# shellcheck disable=SC2044
+for test in $(find "$SCRIPT_DIR" -mindepth 2 -maxdepth 2 -name test.sh | grep "${OZONE_TEST_SELECTOR:-""}" |sort); do
```

Comment:
I think `-mindepth 2 -maxdepth 2` should be added to `test-all.sh`, too, because currently `ozone-mr/hadoop*` tests are executed twice:

```
2020-07-27T10:02:03.8462205Z Executing test in /mnt/ozone/hadoop-ozone/dist/target/ozone-0.6.0-SNAPSHOT/compose/ozone-mr/hadoop27
...
2020-07-27T10:11:13.8415416Z Executing test in /mnt/ozone/hadoop-ozone/dist/target/ozone-0.6.0-SNAPSHOT/compose/ozone-mr/hadoop27
```

Reply:
Hmm. I think I should move out the tricky `find` segment (contains the `#suite` support) to the `testlib.sh` and reuse it from the second level `test.sh`.